extensions/rich_text_components/Skillreview/directives/oppia-noninteractive-skillreview.directive.ts

```diff
@@ -55,9 +55,15 @@ angular.module('oppia').directive('oppiaNoninteractiveSkillreview', [
             // instance, then open the concept card modal. To determine if the
             // RTE is inside a CKEditor instance, check if the offsetParent
             // element contains the data attribute ckeWidgetId.
-            if (event.currentTarget.offsetParent.dataset.ckeWidgetId) {
+            // Also check if the copy mode is enabled within the CKEditor
+            // instance or not. If it is, don't show the RTE modal. The
+            // concept card modal should get triggered instead of the
+            // RTE modal while copying stuff within the copy editor.
+            if (event.currentTarget.offsetParent.dataset.ckeWidgetId ||
+            document.body.style.cursor === 'copy') {
```

Comment:
Again, you can probably separate this into a function to improve the readability.

Reply:
I'll do it.

Reply:
I have made the changes! Have a look.